yt_dlp/extractor/digiview.py

```diff
@@ -0,0 +1,51 @@
+from .youtube import YoutubeIE
+from ..utils import clean_html, int_or_none, traverse_obj, url_or_none, urlencode_postdata
+
+
+class DigiviewIE(YoutubeIE):
+    _VALID_URL = r'https?://(?:www\.)?ladigitale\.dev/digiview/#/v/(?P<id>[0-9a-f]+)'
+    _TESTS = [
+        {
+            # normal video
+            'url': 'https://ladigitale.dev/digiview/#/v/67a8e50aee2ec',
+            'info_dict': {
+                'title': 'Big Buck Bunny 60fps 4K - Official Blender Foundation Short Film',
```

Comment:
These tests are incomplete. Please actually run them.
You should also merge master in to your branch.